index.d.ts

```diff
@@ -368,47 +425,6 @@ declare namespace Emittery {
 		*/
 		eventName?: EventName;
 	}
-
-	/**
-	Async event emitter.
-
-	You must list supported events and the data type they emit, if any.
-
-	@example
-	```
-	import Emittery = require('emittery');
-
-	const emitter = new Emittery.Typed<{value: string}, 'open' | 'close'>();
-
-	emitter.emit('open');
-	emitter.emit('value', 'foo\n');
-	emitter.emit('value', 1); // TS compilation error
-	emitter.emit('end'); // TS compilation error
-	```
-	*/
-	class Typed<EventDataMap extends Events, EmptyEvents extends EventName = never> extends Emittery {
```

Comment:
Is it ok to just remove? It's a breaking change which is maybe ok since emittery is v0 but maybe this could be a deprecated passthrough to the main class?

Reply:
Will try to fix!

Reply:
This is ok. I plan to make this a breaking change.